### DISCUSSION TITLE
go 环境

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#7084] [Q&A]
### 联系方式

iygwei@qq.com

### 1Panel 版本

1.10.20-lts

### 问题描述

我想要安装一个bee 和修改 https://goproxy.cn,direc  要怎么弄 文档没看到

### 重现步骤

https://goproxy.cn,direc

**Comment by wanghe-fit2cloud** [comment]
1Panel 会基于 golang 基础镜像来容器化运行应用，需要根据提示信息指定下图标红的几点即可，如图：
![image](https://github.com/user-attachments/assets/b5bd761a-1f73-4ede-bed7-2ba3dff37b8c)

**Reply by iygwei** [reply]
我是想修改https://goproxy.cn,direc/ 这个元 但是我发现这里无法自己去修改dockerfile

**Reply by iygwei** [reply]
[2Kktadm  | go: github.com/gin-gonic/gin@v1.9.0: Get "https://proxy.golang.org/github.com/gin-gonic/gin/@v/v1.9.0.mod": dial tcp 142.250.198.81:443: i/o timeout
[2Kktadm  | go: github.com/gin-gonic/gin@v1.9.0: Get "https://proxy.golang.org/github.com/gin-gonic/gin/@v/v1.9.0.mod": dial tcp 142.250.198.81:443: i/o timeout
[2Kktadm  | go: github.com/gin-gonic/gin@v1.9.0: Get "https://proxy.golang.org/github.com/gin-gonic/gin/@v/v1.9.0.mod": dial tcp 142.250.198.81:443: i/o timeout
[2Kktadm  | go: github.com/gin-gonic/gin@v1.9.0: Get "https://proxy.golang.org/github.com/gin-gonic/gin/@v/v1.9.0.mod": dial tcp 142.250.198.81:443: i/o timeout
[2Kktadm  | go: github.com/gin-gonic/gin@v1.9.0: Get "https://proxy.golang.org/github.com/gin-gonic/gin/@v/v1.9.0.mod": dial tcp 142.250.198.81:443: i/o timeout

**Reply by wanghe-fit2cloud** [reply]
可以尝试在启动命令中设置环境变量，如：
```
GOPROXY=https://goproxy.cn,direct go run main.go
```
